app/diagnostics.go

```diff
@@ -366,6 +378,13 @@ func trackConfig() {
 	SendDiagnostic(TRACK_CONFIG_ANALYTICS, map[string]interface{}{
 		"isdefault_max_users_for_statistics": isDefault(*utils.Cfg.AnalyticsSettings.MaxUsersForStatistics, model.ANALYTICS_SETTINGS_DEFAULT_MAX_USERS_FOR_STATISTICS),
 	})
+
+	SendDiagnostic(TRACK_CONFIG_ANNOUNCEMENT, map[string]interface{}{
+		"enable_banner":          *utils.Cfg.AnnouncementSettings.EnableBanner,
+		"banner_color":           isDefault(*utils.Cfg.AnnouncementSettings.BannerColor, model.ANNOUNCEMENT_SETTINGS_DEFAULT_BANNER_COLOR),
```

Comment:
Can we call this `isdefault_banner_color` so it's consistent with other settings